content/api/recipient-lists.apib

```diff
@@ -11,52 +11,81 @@ transmission data, or a stored recipient list id attribute can be specified.
 
 The Recipient List API operates on lists as a whole and does not currently support management of individual recipients.
 
-## Recipient List Attributes
-| Field         | Type     | Description                           | Required   | Notes   |
-|------------------------|:-:       |---------------------------------------|-------------|--------|
-|id |string       |Short, unique, recipient list identifier |no  |If an id is not specified, one is generated.  Maximum length - 64 bytes |
-|name    |string       |Short, pretty/readable recipient list display name, not required to be unique |no  |If a name is not specified, then defaults to the same value as id.  Maximum length - 64 bytes |
-|description |string |Detailed description of the recipient list|no | Maximum length - 1024 bytes|
-|attributes |JSON object  |Recipient list attribute object  |no  |This JSON object allows users to store arbitrary metadata related to this list.  This data is not used by the API.  It is only for the user. |
-|recipients |JSON array  |Array of recipient objects |yes |For a full description, see the [Recipient Attributes](#header-recipient-attributes). |
-
-### Recipient Attributes
+### Recipient List Object
+
++ Data Structure: Attributes
+    + id (string) - Short, unique, identifier. If an id is not specified, one is generated. <br /> Maximum length: 64 bytes
```

Comment:
Just "Unique alphanumeric ID", leave the "If an id is .." to the POST description